src/esp/gfx/RenderTarget.cpp

```diff
@@ -145,10 +145,14 @@ struct RenderTarget::Impl {
         .draw(depthUnprojectionMesh_);
   }
 
-  void renderEnter() {
+  void renderEnter(const Cr::Containers::Optional<Mn::Color4>& clearColor) {
     framebuffer_.clearDepth(1.0);
     if (flags_ & Flag::RgbaBuffer) {
-      framebuffer_.clearColor(0, Mn::Color4{0, 0, 0, 1});
+      if (clearColor) {  // custom clear color
+        framebuffer_.clearColor(0, *clearColor);
+      } else {  // default clear color
+        framebuffer_.clearColor(0, Mn::Color4{0, 0, 0, 1});
+      }
```

Comment:
What about, instead of the optional and the extra branch, have `Mn::Color4{0, 0, 0, 1}` as a default value for the clear color argument?

Reply:
One additional option is to pass a reference to the sensor for a given render target.  The render targets are per sensor and this would let it just grab the clear color from the spec.  This would also cleanly resolve the double clear issue pointed out on the python side.